examples/plot_simulate_gamma.py

```diff
@@ -33,30 +33,49 @@
 net = Network(params)
 dpls = simulate_dipole(net, n_trials=1)
 
+###############################################################################
+# The network requires some time to reach steady state. Hence, we omit the
+# first 50 ms in our time-frequency analysis.
+
+tstart = 50
+mask = dpls[0].times > tstart
+times = dpls[0].times[mask]
+data = dpls[0].data['agg'][mask]
+
 ###############################################################################
 # We can plot the time-frequency response using MNE
 import numpy as np
 import matplotlib.pyplot as plt
-from mne.time_frequency import tfr_array_multitaper
+from mne.time_frequency import tfr_array_morlet
 
 fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6))
-dpls[0].plot(ax=axes[0], layer='agg', show=False)
+axes[0].plot(times, data)
 
 sfreq = 1000. / params['dt']
-time_bandwidth = 4.0
 freqs = np.arange(20., 100., 1.)
-n_cycles = freqs / 4.
+n_cycles = freqs / 8.
```

Comment:
Is this to keep a consistent time resolution between morlet and multitaper? I'm struggling to remember exact impact for the two methods.

Reply:
no, not really. MNE complains about wavelet cycles being too long if n_cycles = freqs / 4. Not sure why it doesn't do the same with multitaper. Anyhow I think [HNN-GUI uses `n_cycles = freqs / 7`](https://github.com/jonescompneurolab/hnn/blob/330677d06b40813c46c0cd7b1bfb701566efdddf/specfn.py#L64) but MNE doesn't allow that, so I picked the closest possible :)

As for the difference between the two, [this page](https://www.fieldtriptoolbox.org/tutorial/timefrequencyanalysis/#multitapers) gives a nice summary. Essentially, morlet is like doing multitaper with a single Gaussian taper

Reply:
> Not sure why it doesn't do the same with multitaper.

maybe an opportunity to make a small PR to MNE if you are willing to dig into it? :)